packages/scandipwa/public/index.html

```diff
@@ -27,6 +27,8 @@
         // do reverse sort in order prevent an issue like store code `en` replaces store code `en_us`
         window.storeList = ['default'].sort().reverse();
         window.storeRegexText = `/(${window.storeList.join('|')})?`;
+
+        window.website_code = "";
```

Comment:
Will it work if `website_code` is empty? Did you check it?
I'd suggest giving it the default "base" value instead.

Reply:
No it didn't work :d was empty string as key obviously, changed it to 'base'